### PR TITLE
refactor(event/eventfilter and constants/eventfilterconstants): refactored component

### PR DESCRIPTION
**Pull request by Vladosik12333**
## EventsExpressFrontend

* [Jira task](https://localhost-vlad.atlassian.net/jira/software/projects/EEF/boards/3?selectedIssue=EEF-24)

## Code reviewers

- [ ] All

### Second Level Review

- [ ] @andrii-kuzmichov || @VadymKharyton 

## Summary of issue

Refactored component and add constants for component.

## Summary of change

event/eventFilter and constants/eventFilterConstants

## Testing approach

-

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/components/event/EventFilter/Event-filter.js [hunk 1]
```diff
@@ -35,14 +51,9 @@ class EventFilter extends Component {
...
-        keyWord: initialValues.keyWord,
-        dateFrom: initialValues.dateFrom,
-        dateTo: initialValues.dateTo,
-        categories: initialValues.categories,
-        statuses: initialValues.statuses,
-        radius: initialValues.radius,
+        ...initialValues,
         selectedPos:
-          initialValues.selectedPos != null
+          initialValues.selectedPos !== null
```
Comment (VadymKharyton) [inline src/components/event/EventFilter/Event-filter.js]: Simplify to `initialValues.selectedPos ? ... : ...`

File: src/components/event/EventFilter/Event-filter.js [hunk 2]
```diff
@@ -52,157 +63,165 @@ class EventFilter extends Component {
...
+          <div className="form-group">
+            <Field
+              name="keyWord"
+              component={renderTextField}
+              type="input"
+              label="Keyword"
+            />
+          </div>
+          {viewMore && (
+            <>
```
Comment (VadymKharyton) [inline src/components/event/EventFilter/Event-filter.js:99]: Please check if we can remove fragment tag

File: src/components/event/EventFilter/Event-filter.js (hunk 1 above)
Comment (Vladosik12333) [reply src/components/event/EventFilter/Event-filter.js]: ++

File: src/components/event/EventFilter/Event-filter.js (hunk 2 above)
Comment (Vladosik12333) [reply src/components/event/EventFilter/Event-filter.js:99]: No, we can`t.

**Review by VadymKharyton — APPROVED** [review]

**Pull request merged**